### DISCUSSION TITLE
Windows App SDK name? if it superseeds UWP

### DISCUSSION THREAD

**Discussion by AathifMahir** [microsoft/WindowsAppSDK#1620] [General] [answered]
UWP naming is mainly became viable since it's supports Windows, Xbox, etc..

But When it comes to Windows App SDK, as name refers it's mainly for Windows App Development Only

In this case, what happens if Windows App SDK supports Xbox and other Stacks in the future.

If Windows App SDK supports Xbox in that case name wouldn't make sense right?

Any opinion on this

Edit: As far as recent announcement regarding UWP support, Windows App SDK is the Recommended path forward, Hope Windows App SDK would become truly universal that supports all the MS Stacks in that case Windows App SDK naming doesn't make sense

**Comment by dotMorten** [comment, marked as answer]
The W in UWP is also "Windows", and XBox, HoloLens, Surface Hub etc are all Windows kernels too, so I don't really get the issue with the name, if those (huge) "what ifs" ever actually became true.

**Comment by saint4eva** [comment]
> UWP naming is mainly became viable since it's supports Windows, Xbox, etc..
> 
> But When it comes to Windows App SDK, as name refers it's mainly for Windows App Development Only
> 
> In this case, what happens if Windows App SDK supports Xbox and other Stacks in the future.
> 
> If Windows App SDK supports Xbox in that case name wouldn't make sense right?
> 
> Any opinion on this
> 
> Edit: As far as recent announcement regarding UWP support, Windows App SDK is the Recommended path forward, Hope Windows App SDK would become truly universal that supports all the MS Stacks in that case Windows App SDK naming doesn't make sense

It is not going to support Xbox and HoloLens, so do not bother yourself.

**Comment by GreenShadeZhang** [comment]
I agree with you
